packages/rspack/src/exports.ts

```diff
@@ -45,7 +45,7 @@ export const WebpackError = Error;
 
 export type { Watching } from "./Watching";
 
-const sources = require("webpack-sources"); // use require to avoid wrong types, @types/webpack-sources is outdate
+import sources = require("webpack-sources");
```

Comment:
Can we use `import ... from` here?

Reply:
noop. it will enforce some problems.

The `webpack-sources.d.ts` file uses ESM to declare the module, while `webpack-sources` is implemented with CommonJS.

Using ESM import will cause misalignment between the source and the types defined.

If we declare a namespace and export default in `webpack-sources.d.ts` to fit CommonJS source, `api-extractor` will not support it.

Therefore, use `import sources = require("webpack-sources");` here.